src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpRawlsDAO.scala

```diff
@@ -26,6 +30,21 @@ import scala.concurrent.{ExecutionContext, Future}
 class HttpRawlsDAO( implicit val system: ActorSystem, implicit val executionContext: ExecutionContext )
   extends RawlsDAO with RestJsonClient {
 
+  // TODO: Copied from HttpOntologyDAO. How to refactor to a common trait ???
```

Comment:
Maybe put it in `RestJsonClient`?
```
private def getHostConnectorSetup(address, port, scheme): HostConnectorSetup = HostConnectorSetup(...)

protected final def getHostConnector(address, port, scheme): Future[ActorRef] = {
  implicit val timeout:Timeout = 60.seconds // timeout to get the host connector reference
  for (Http.HostConnectorInfo(connector, _) <- IO(Http) ? getHostConnectorSetup(address, port, scheme)) yield connector
}
```
Just a thought. Then any subclass could potentially build their own connector for async operations.